client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -58,8 +58,18 @@ export const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId,
 	const { __ } = useI18n();
 	const dispatch = useDispatch();
 	const queryClient = useQueryClient();
-	const { data: hasValidQuota } = useHasValidQuotaQuery( siteId );
 	const [ loadingError, setLoadingError ] = useState( false );
+	const [ loadingValidQuotaError, setLoadingValidQuotaError ] = useState( false );
```

Comment:
@kozer, What do you think about this variable name?

```suggestion
	const [ isValidQuotaError, setIsValidQuotaError ] = useState( false );
```